app/screens/create-competition/page.tsx

```diff
@@ -0,0 +1,19 @@
+import Link from "next/link";
+import React from "react";
+
+type Props = {}
```

Comment:
I believe interfaces have better ts support, I recommend them when possible, but there are times you need to use type.

this explains it well
[https://www.typescriptlang.org/play#example/types-vs-interfaces](url)

Reply:
Yup use interfaces instead of types